### ISSUE TITLE
Reducing tailwindcssProperties.json contribution to production bundle size

### ISSUE THREAD

**Issue by HolySamosa** [richardgill/tailwind-override#10]
`tailwindcssProperties.json` is a significant contributor (~73 kB) to the size of our production bundle.  Any ideas of how this might be reduced-- perhaps a path towards optimization in a future release?

**Comment by richardgill** [comment]
Hey @HolySamosa, I hadn't considered that the json file was pretty big. The file that ships with the package is ~116KB. If you minify it and remove all the whitespace it's ~73KB as you say. 

The code for this project is ~2KB, but this file was unconditionally imported that meant the size of the package was ~118KB or maybe ~75KB depending on how your bundler (e.g. webpack) is configured.

I've just released version `0.1.0` which allows users to import either: `tailwind-override` (~118KB) or `tailwind-override/lib/core` which is ~2KB, but you need to bring your own `tailwindcssProperties.json` as an object in javascript which might make it easier for you to solve this issue yourself.

If you use gzip on the ~116KB file it comes out at ~8.7KB.

The way the file is built is quite verbose, and could probably be shortened, but it would make the implementation of this module much more complicated.

**Comment by richardgill** [comment]
@HolySamosa sorry for the delay, but I've got this working in my project.

First I ran: 

`NODE_ENV=production npx tailwindcss-cli@latest -o ./src/generated/temp/tailwindOutput.css`
Which produces my a css files with only the classes I'm actually using.

Then I produce the json file for those classes:
`node_modules/.bin/tailwind-override --inputFile ./src/generated/temp/tailwindOutput.css --outputFile ./src/generated/temp/tailwindProperties.json`

Finally in create-react-app I imported it like this: 


```
import { overrideTailwindClasses } from 'tailwind-override/lib/core'

let tailwindProperties = {}

if (process.env.NODE_ENV === 'production') {
  import('../generated/temp/tailwindProperties.json').then(tailwindPropertiesImport => {
    tailwindProperties = tailwindPropertiesImport
  })
} else {
  import('tailwind-override').then(tailwindOverrideImport => {
    tailwindProperties = tailwindOverrideImport.tailwindProperties
  })
}

export const tailwindOverrideClasses = (classes: string) => {
  if (tailwindProperties === {}) {
    throw new Error('empty tailwind properties was provided')
  }
  return overrideTailwindClasses(classes, { tailwindProperties })
}
```

This should solve your problem.
